### ISSUE TITLE
"Back to default zoom" feature

### ISSUE THREAD

**Issue by druxstr** [umap-project/umap#300]
Hi,

Enhancement suggestion : the "Home icon" should send you back to the default zoom of the map you're browsing. Not the homepage of the instance.

Many users told me they tried to go back to default zoom by clicking on the "Home icon" and where confused by it's behavior.

But even if the "Home icon" doesn't change, a "back to default zoom" feature would be nice.

Cheers

**Comment by yohanboniface** [comment]
Nice suggestion. Not sure though, at the icon level, how to make those two actions clear enough (back to home and back to default view). But it's worth a try.

**Comment by druxstr** [comment]
I agree that finding a good icon for this isn't easy. Someting inspired by [this kind of meeting point logo](https://commons.wikimedia.org/wiki/File:Logo_Haus_des_Freien_Wissens_und_Community_St%C3%BCtzpunkte_schwarzweiss.svg) *may* do the trick. With arrows pointing to a circle / point, or pointing in the opposite direction maybe. 

One way to make the 2 actions clearly different could be to place the "Default zoom" icon between the "Zoom in" (+) and "Zoom out" (-) buttons. Or just after these 2 buttons, before the layer icon. 

Essentially, to NOT put it with the icons displaying when you click "More". But to place it alongside the default displaying icons. 

This way the "Default zoom" and "Go to home page" actions wouldn't be simultaneously revealed to the user. And as "Default zoom" is a zoom related action, it could make sense to place it near the zoom in/out actions.

**Comment by yohanboniface** [comment]
@Aurelie-Jallut something we can have a look together ? :)

**Comment by Aurelie-Jallut** [comment]
@yohanboniface yes :) the zoom + / zoom - / full screen / default zoom icons could be grouped together.
